crates/languages/src/ruby/injections.scm

```diff
@@ -0,0 +1,4 @@
+(heredoc_beginning)
+(heredoc_body
+  (heredoc_end) @language) @content
+(#downcase! @language)
```

Comment:
I think you only need lines 2-3:

```
(heredoc_body
  (heredoc_end) @language) @content)
```

The first line matches a `heredoc_beginning`, but doesn't capture anything. The last line doesn't seem valid, as it only contains a predicate, but no actual pattern.

Reply:
The content had to be after the heredoc_body in my tests. Did you succeed with the example you gave ?

This is actually quite close to the example from tree-sitter so I'm quite confident it is ok.

The example:

```
(heredoc_body
  (heredoc_end) @injection.language) @injection.content
```

Also there's an extra parenthesis in your code I think :/

Also it is valid in the sense that is works, I've been using it for some time now :)

Reply:
Just to rephrase - the first and last lines don't do anything, and definitely are not needed. Can you remove them?

Reply:
got you, done!